vshard/storage/init.lua

```diff
@@ -2493,6 +2493,11 @@ local function route_dispenser_create(routes)
     return {
         rlist = rlist,
         map = map,
+        -- If any of uuid was skipped, then some serious
+        -- error occurred on the receiver, we should say
+        -- the user that rebalancer_apply_routes failed
+        -- in such case
+        is_uuid_skipped = false
```

Comment:
Isn't it's better to have a map of all errors, which led to skipping an uuid? We cannot save just only first or last occurred error, as they are most likely not related to each other

Reply:
Well, you can save many, but you can expose to the server info only one. What is the point in saving them all then? Yes, they are unrelated most likely, but usually they happen repeatedly. It is enough to expose one (when user will fix it, it will see the next one), and log all of them (it is already done AFAIK).